ipapython/ipautil.py

```diff
@@ -142,6 +153,7 @@ def __init__(self, addr, match_local=False, parse_netmask=True,
 
         if isinstance(addr, CheckedIPAddress):
             self.prefixlen = addr.prefixlen
+            self._net = addr._net
```

Comment:
Given `addr` is an instance of `CheckedIPAddress`, when this is called from `CheckedIPAddress.__init__`:
```python
        super(CheckedIPAddress, self).__init__(addr)
```
it will call this bit (and only this bit) of `UnsafeIPAddress.__init__`:
```python
        if isinstance(addr, UnsafeIPAddress):
            self._net = addr._net
            super(UnsafeIPAddress, self).__init__(addr,
                                                  flags=self.netaddr_ip_flags)
            return
```
which uncoditionally copies the value of `addr._net` to `self._net`, so how exactly would `self._net` end up as `None` when `addr._net` is not `None` too?